containers/kubernetes/how-to/create-cluster.mdx

```diff
@@ -27,27 +27,33 @@ To administrate your Kubernetes Cluster easily, Scaleway provides a `.kubeconfig
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
 </Message>
 
-This document concerns the creation and management of a **Kubernetes Kapsule** cluster. To create a Kubernetes Kosmos Cluster, refer to the [alternative documentation](/containers/kubernetes/how-to/create-kosmos-cluster)
-
-1. Click **Kubernetes** in the **Containers** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Kubernetes Kapsule dashboard displays.
-2. Click **Create cluster**. The first page of the cluster creation wizard displays. This concerns the configuration of your cluster.
-3. Complete the following steps of the wizard:
-    <Lightbox src="scaleway-choose-cluster-type.webp" alt="" />    
-    
-    * Choose a **Cluster Type**. This can be Kubernetes Kapsule or Kubernetes Kosmos.
-      * **Kubernetes Kapsule** allows you to create clusters exclusively of Scaleway Instances, and includes features such as node auto-healing and pool auto-scaling.
-      * **Kubernetes Kosmos** allows you to attach an Instance or dedicated server from any Cloud provider to a Scaleway Kubernetes control plane.
+1. Click **Kubernetes** in the **Containers** section of the side menu. The Kubernetes Kapsule dashboard displays.
```

Comment:
Just out of interest, any particular reason to take out the link to the console?